### ISSUE TITLE
Incompatibility with some other seal mods.

### ISSUE THREAD

**Issue by FloridaMen** [AxBolduc/GreenSeal#3]
The XMult ability of the green seal breaks some other seals if they don't specify a mult value. Attached is a version of greenSeal.lua that is fixed.
[GreenSeal.txt](https://github.com/AxBolduc/GreenSeal/files/15083378/GreenSeal.txt)

**Comment by FloridaMen** [comment]
Of course, you must change the extension from .txt to lua; github doesn't allow .lua uploads in issue reports.

**Comment by JonJaded** [comment]
Thanks!
